### DISCUSSION TITLE
Single Blocks / Raw JSON

### DISCUSSION THREAD

**Discussion by chris-schra** [payloadcms/payload#710] [Feature Requests & Ideas]
We're moving from Strapi and stored some custom data as raw JSON.
I'd prefer to use Blocks for that, but it seems there's no such thing as a "single block" - like, pages should have a attribute metadata which is just a JSON for us to store, and it will always be a 1:1 (or 1:null) relationship

Edit: I noticed I can use type:"richText" for that, but it feels a bit dirty :D

**Comment by chris-schra** [comment]
urgh, I'm so sorry
https://payloadcms.com/docs/fields/group

**Reply by jmikrut** [reply]
No worries! Sounds like you found what you need. If not, let us know and we'll be around to help!

**Reply by revnelson** [reply]
Is there any way to use a Block as a Group? For example, I have a "Hero" block that has options to populate various hero sections with various styles on our website. I love being able to insert this into the lexical editor for more flexible layouts. However, sometimes I want to build a more structured layout for a page and want to explicit place a single "Hero" in the fields. I don't want to just put a 'richText' field and trust whoever may edit the document later to put in the correct block.
